src/cpuprofiler.js

```diff
@@ -15,7 +15,7 @@
 // That doesn't work for Chrome in turn, so need to resort to user agent
 // sniffing.. (sad :/)
 if (!performance.realNow) {
-  var isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
+  var isSafari = typeof navigator !== "undefined" && /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
```

Comment:
nit: we use single quotes in JS files.

otherwise lgtm!  Thanks for working on this.

Reply:
fixed !